zulipterminal/model.py

```diff
@@ -344,6 +347,15 @@ def update_stream_message(self, topic: str, msg_id: int,
         display_error_if_present(response, self.controller)
         return response['result'] == 'success'
 
+    def fetch_custom_emojis(self) -> Dict[str, Dict[str, str]]:
```

Comment:
We could `TypedDict` the emoji types? There's even scope for using `Literal` here given the limited set of emoji-types, though we need to understand the realm vs custom vs zulip-extra.